api-server/models.py

```diff
@@ -36,6 +35,7 @@ class NoSelectedSourceCodeError(Exception):
 class Session(models.Model):
     id = fields.CharField(max_length=20, pk=True)
     created_at = fields.DatetimeField(auto_now_add=True)
+    version = fields.IntField(default=0)
```

Comment:
what is version?

Reply:
I kept it there to debug any "concurrent" updation issues.